spec/features/authentication_spec.rb

```diff
@@ -1,28 +1,55 @@
 require 'rails_helper'
 
-describe "the signin process", :type => :feature do
-  let(:user) { User.create(:email => 'user@example.com', :password => 'password') }
+describe 'Authentication' do
+  let(:user) { create(:user) }
+  let(:admin) { create(:admin) }
 
-  scenario "Signs me in" do
-    visit root_path
-    click_link 'Login'
-    fill_in 'Email', with: user.email
-    fill_in 'Password', with: user.password
-    click_button 'Log in'
-    expect(current_path).to eq(root_path)
-    expect(page).to have_content('Signed in successfully')
+  context 'Guest user' do
+    scenario "Can sign up" do
+      visit new_user_registration_path
+      fill_in "Email", with: 'guest@example.com'
+      fill_in "Password", with: 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.